docs/NewFeatures/lifecycle_expiration_policies.rst

```diff
@@ -0,0 +1,68 @@
+=============================
+
+Cloud users can apply lifecycle expiration rules (specified in Amazon’s
+`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+to buckets managed through Zenko. These rules are triggered after a defined
+time has passed since the object’s creation. Zenko supports expiration of
+versioned or non-versioned objects, when a defined number of days has
+passed since those objects’ creation. This enables automatic deletion of
+older versions of versioned objects to reclaim storage space.
+
+Several Zenko components are used in lifecycle management. Most rely on Kafka
+and ZooKeeper to pass state to each other.
+
+Bucket lifecycle characteristics inhere to each bucket. Zenko’s lifecycle
+management feature enforces, but does not set these characteristics. When
+lifecycle expiration is enabled, the host cloud enforces buckets’ lifecycle
+rules. If CRR operation is enabled, Zenko replicates the expiration rules to

```

Comment:
"If CRR operation is enabled, Zenko replicates the expiration rules to all backup clouds"

This is how it is currently, but this is more of a bug that needs to be addressed, as AWS does not replicate lifecycle actions, neither do we want this behavior for Zenko. So better not to mention this as we plan to fix the current behavior soon enough.

Reply:
done.